Doc/library/math.rst

```diff
@@ -213,6 +213,14 @@ Number-theoretic and representation functions
    of *x* and are floats.
 
 
+.. function:: nextafter(x, y)
+
+   Return the next floating-point value after *x* towards *y*.
+
+   If *x* equals to *y*, return *y*.
```

Comment:
Suggest "is equal to" in place of "equals to"; it reads a bit better.

Reply:
Done. You may suggest this change to the Linux manual page of nextafter() as well where I stole this sentence :-)